robot/power.py

```diff
@@ -7,6 +8,18 @@
 LOGGER = logging.getLogger(__name__)
 
 
+# Keep this in sync with `robotd`
+class PowerOutput(enum.Enum):
+    """An enumeration of the outputs on the power board."""
```

Comment:
We should validate that the name -> id mapping here is correct (I'm not sure the best way to do that other than getting a power board and trying it).

Reply:
(this was checked by @RealOrangeOne)